data/data/manifests/openshift/cloud-creds-secret.yaml.template

```diff
@@ -16,8 +16,6 @@ metadata:
   name: vsphere-creds
 {{- else if .CloudCreds.Ovirt}}
   name: ovirt-credentials
-{{- else if .CloudCreds.Nutanix}}
```

Comment:
Can you elaborate on why credentials are no longer required in the cluster?

Reply:
Nutanix supports only the credentials manual mode. Users have to use the ccoctl cli tool to help generate the nutanix-credentials secret yaml files, at cluster installation time and later when need to update the nutanix-credentials for any existing cluster. The installer does not need to generate an nutanix-credentials secret yaml files in the "kube-system" namespace.